server/app/api/v1/endpoints/twilio_webhooks.py

```
@@ -0,0 +1,214 @@
+        logger.error(f"❌ Error processing inbound SMS webhook: {e}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Failed to process webhook"
+        )
+
+
+@router.get("/test-status")
+async def test_status_webhook(
+    message_sid: str,
+    status: str,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)